diff --git a/node_modules/react-native-reanimated/lib/typescript/index.web.d.ts b/node_modules/react-native-reanimated/lib/typescript/index.web.d.ts
index 70b732e..33bbf1b 100644
--- a/node_modules/react-native-reanimated/lib/typescript/index.web.d.ts
+++ b/node_modules/react-native-reanimated/lib/typescript/index.web.d.ts
@@ -1,2 +1,3 @@
+import * as Animated from './Animated';
 export * from './reanimated2';
-export * as default from './Animated';
+export default Animated;
